### PR TITLE
Addition of the eQ header

### PR DESCRIPTION
**Pull request by armstrongb**
### Context of this PR

- Addition of current `header` code from eQ and addition of `_header2.scss` to bring the pattern library up to parity with ’eQ Runner‘.
- It also required amendments to the bar component for inclusion in header.

### How to review this PR

- Checkout the `component-eq-header` branch and a run `yarn start`.
- Navigate to the header component `/components/detail/header` and review each of the component files.

### CODE REVIEW

**Comment by armstrongb** [comment]
Forms part of closing issue #9

File: components/header/_header2.scss [hunk 1]
```diff
@@ -0,0 +1,65 @@
+$header2-bp: 36em;
+
+.header2 {
```
Comment (hamishtaplin) [inline components/header/_header2.scss]: This should be named something more meaningful than `.header2`. Are we even keeping the old header around? If so, it's probably a modifier of the existing `.header` eg. `.header--new`

File: components/header/_header2.scss [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  @include mq($header2-bp) {
+    margin-bottom: 0;
+    flex: 1 1 100%;
+  }
+}
+
+.header2__info {
+  white-space: nowrap;
+  min-height: 3rem;
+  width: 550px;
```
Comment (hamishtaplin) [inline components/header/_header2.scss]: What's the reason for using `px` here?

File: components/header/_header2.scss [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+.header2 {
+  display: block;
+  overflow: hidden;
+  padding: 1.4rem 0 1rem;
+  &::before {
+    content: "";
+    display: block;
+    width: 100%;
+    height: 8px;
```
Comment (hamishtaplin) [inline components/header/_header2.scss]: Should this be in `rems`?

File: components/header/header.hbs [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<header class="page__header">
```
Comment (hamishtaplin) [inline components/header/header.hbs]: Seems like this template is crossing concerns.. `page__header` is part of a `page` module.

File: components/header/_header2.scss (hunk 1 above)
Comment (saedwards) [reply components/header/_header2.scss]: Have reverted the class name to remove class references with '2' in them. This doesn't appear to be a modifier but a fix of the original header.

File: components/header/_header2.scss (hunk 3 above)
Comment (saedwards) [reply components/header/_header2.scss]: Are you referring to just the height? Should be a straight forward fix. Not sure that rems can't directly substitute percentages.

File: components/header/_header2.scss (hunk 2 above)
Comment (saedwards) [reply components/header/_header2.scss]: I can't find any references to this header element in eq and ras frontstage--do you know if we still need this?

File: components/header/header.hbs (hunk 4 above)
Comment (saedwards) [reply components/header/header.hbs]: Agree, have removed the page__header from the header template. This should be used in the Page component only.

**Comment by saedwards** [comment]
Addition of header__info that was included in CSS but not in the template

File: components/header/_header2.scss (hunk 2 above)
Comment (saedwards) [reply components/header/_header2.scss]: Looking at this in the browser, not sure why we need a width at all.

Comment (saedwards) [reply components/header/_header2.scss]: Decision to remove this element after talks with @armstrongb. Is no longer used

**Pull request merged**